components/FeatureHighlight/BUILD

```diff
@@ -92,7 +104,8 @@ mdc_objc_library(
         ":FeatureHighlight",
         ":ColorThemer",
         ":FontThemer",
-        ":private",
+	":FeatureHighlightAccessibilityMutator",
+       	":private",
```

Comment:
Nit/formatting: Please remove tab characters and only use spaces.

Reply:
Done.